### PR TITLE
add_credential_setup_for_google_map_api_token

### PR DESCRIPTION
**Pull request by Zhi0105**
## Summary
- Add controller to access google api token from back end to front end

## Screenshot
google key

### CODE REVIEW

File: app/controllers/api/v1/google_keys_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Api::V1::GoogleKeysController < ApplicationController
+  def index
+    @google_key = Rails.application.credentials.google_api_token[:token]
+    render json: {
+      data: {
+        key: @google_key
+      }
+    }
+  end
+end
```
Comment (analyn-cajocson) [inline app/controllers/api/v1/google_keys_controller.rb:10]: what is the use of this API?

Comment (Zhi0105) [reply app/controllers/api/v1/google_keys_controller.rb:10]: For google map sa front end ma'am, para di po nakalabas sa code, nilagay ko nalang po sa credential ng back end

Comment (Zhi0105) [reply app/controllers/api/v1/google_keys_controller.rb:10]: > what is the use of this API?

For google map sa front end ma'am, para di po nakalabas sa code, nilagay ko nalang po sa credential ng back end

Comment (analyn-cajocson) [reply app/controllers/api/v1/google_keys_controller.rb:10]: Yah but you're actually exposing your token to the public. Maybe you can try other implementation

Comment (Zhi0105) [reply app/controllers/api/v1/google_keys_controller.rb:10]: > Yah but you're actually exposing your token to the public. Maybe you can try other implementation

okay po ma'am try po ako other way

**Pull request closed without merging**